process_pr.py

```diff
@@ -1090,7 +1090,7 @@ def process_pr(
         if (
             repo.full_name == "cms-sw/cms-bot"
             and os.getenv("CMS_BOT_TEST_BRANCH", "master") == "master"
-            and False
+            and pr.state != "closed"
```

Comment:
change looks reasonable. I am not sure if check for "pr.state == "open"` is better/safer

Reply:
that I know, just wondering if there was any reason to use `!=` instead of `==` ( just for my education :-) ) ?